### ISSUE TITLE
Nodejs.org redesign meeting refresh

### ISSUE THREAD

**Issue by hackygolucky** [nodejs/nodejs.org#1338]
Since I've been at the Foundation(March 2016), I've heard from many parts of the project, members, and Board directors that the current nodejs.org needs some love. I was pointed to the discussions and work happening for the prior plan of redesign(such as #1074) and asked to hold off to let that work be seen through. 

I think it has been along enough that it would be good for relevant parties to get together in a meeting, check-in, and see if it is time for iterating on the plan in place. 

@nodejs/website  @nodejs/community-committee @leo 

### Please indicate your availability [HERE](https://doodle.com/poll/82ddcmbbq5hka96e).

Agenda TBD
- Review the prior plan(can someone link this please?)
- list an item here that you think is vital to this meeting
- another item here

**Comment by Trott** [comment]
I wonder if it would make sense to spend some Foundation money to get some professional UX research done to improve the site. No disrespect to the wonderful people who voluntarily maintain the site, but collecting usable data about UX can be difficult and is often costly.

**Comment by hackygolucky** [comment]
@Trott Yeah I think this is definitely something we should talk about in the session. We have -so- many users, and if we look at the amount of traffic we get, I think we owe it to them to get a quality dataset from which we base future decisions. If we determine this could be helpful, we can discuss it with the Board.

**Comment by raphaelokon** [comment]
@Trott Totally agree. @hackygolucky Absolutely!

**Comment by bnb** [comment]
@hackygolucky AFAIK there was no published plan - it was mostly context between Mikeal and Leo. This is the repo that was used to begin feedback intake:

https://github.com/nodejs/nodejs.org-new

Additionally, these are the relevant issues from that repo:
https://github.com/nodejs/nodejs.org-new/issues/4
https://github.com/nodejs/nodejs.org-new/issues/5

And feature request issues:
https://github.com/nodejs/nodejs.org-new/issues/1
https://github.com/nodejs/nodejs.org-new/issues/2

**Comment by bnb** [comment]
Also, from https://github.com/nodejs/nodejs.org-new/issues/4 it seems @evilrabbit is involved, and should be welcomed and invited to the meeting as well, if they can make it :)

**Comment by nebrius** [comment]
I'm going to be off the grid during this time and so I cannot attend. That said, I don't think it's all that necessary for me to be there?

**Comment by bnb** [comment]
@nebrius Don't thing you're required. We'll have at least two members from the CommComm (probably more), so I think we should be set 👍

**Comment by hackygolucky** [comment]
@bnb @pup @pierreneter @Trott  I haven't scheduled this yet because I'm still waiting on a few of the folks who were working on the redesign to chime in on availability. 

@bnb could you @ them here to boost?

**Comment by Fishrock123** [comment]
I should have time to do work on this on a technical / implementation end, when it becomes a thing. I have filled out the doodle.

**Comment by bnb** [comment]
@hackygolucky The people working on the redesign were @leo and @evilrabbit - hopefully this message gets to them 🙏

**Comment by leo** [comment]
I'd also love to work on writing it in [Next.js](https://github.com/zeit/next.js). Talking to @evilrabbit regarding the design... 😊

**Comment by fhemberger** [comment]
@leo Why would we need server side rendered react.js for a entirely static website? 🤔 

I'd love to keep our technical setup with metalsmith, the website content needs to be easily translatable also for non-technical contributors.

**Comment by leo** [comment]
Every website nowadays needs its first load to be rendered on the server. Otherwise you're causing a white screen until the client side assets are downloaded (read more [here](https://m.youtube.com/watch?v=evaMpdSiZKk)). After that has happend, the client side will pick up.

It's should be just as easy to translate and much cleaner (because of the React components).

**Comment by leo** [comment]
Also Next.js is not just about SSR, it's about providing the cleanest, most straight-forward and future-proof toolchain for building websites and apps in 2017.

https://zeit.co/docs/examples/next

**Comment by fhemberger** [comment]
We *are* already rendering everything on the server. As static HTML. The page size is so small that you don't get any benefit from client side rendering afterwards. It's a website. Not a web application.

Also I bet HTML is still the cleanest, most straight-forward and future-proof way to create a website.

**Comment by pierreneter** [comment]
So, we just need to redesign the template as well as problems with UX for the site `or` do it and create a new source code for this site with NEXT.js or react.js?

My vote for `react.js`, we can solve the problem of a `white screen` with a `loading screen page` if necessary.

**Comment by hackygolucky** [comment]
Okay I'm the worst and had waited for some folks to schedule in the doodle, and then the days passed that we had chimed in for. So!

If y'all could chime in again [HERE](https://beta.doodle.com/poll/82ddcmbbq5hka96e), I promise I'll schedule it this go round. cc @pierreneter @Fishrock123 @bnb @leo @nebrius @raphaelokon @leo

**Comment by fhemberger** [comment]
Closing in favor of #1534.
